src/templates/Brochure/BrochureTemplate.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { FunctionComponent } from "react";
+import { TemplateProps } from "@govtechsg/decentralized-renderer-react-components";
+import { getDocumentData } from "../../utils";
+import { Wrapper } from "../../core/Wrapper";
+import { BrochureSchema, BrochureDocument } from "./types";
+import styled from "@emotion/styled";
```

Comment:
i was intending to remove emotion from this repo eventually, if possible try to adopt tailwind?